tasks/4.js

```diff
@@ -0,0 +1,25 @@
+var arr = [1, 2, 1, 3, 4];
+
+function mapArr(arr) {
+  var newArr = [];
+  arr.map(function (x) {
```

Comment:
You are using `map` which should return all of the array items mutated, but the callback function does filtering, in such case a `forEach` would be a more correct solution.